src/node_file.cc

```diff
@@ -831,6 +831,7 @@ static void Symlink(const FunctionCallbackInfo<Value>& args) {
   if (req_wrap != nullptr) {  // symlink(target, path, flags, req)
     AsyncDestCall(env, req_wrap, args, "symlink", *path, path.length(), UTF8,
                   AfterNoArgs, uv_fs_symlink, *target, *path, flags);
+    req_wrap->SetReturnValue(args);
```

Comment:
It is not safe to call `req_wrap->SetReturnValue(args)` like this without checking if the pointer returned by `AsyncDestCall` is `nullptr`, see https://github.com/nodejs/node/pull/18811 (that PR just calls `SetReturnValue ()` inside `AsyncDestCall` so the binding do not have to touch `req_wrap`)

Reply:
I see, thank you for information!